_pages/define/problem.md

```diff
@@ -1,51 +1,56 @@
 ---
+layout: post
 title: Define the problem
-permalink: /future/problem/
+permalink: /define/problem/
+sidenav: define
+sticky_sidenav: true
 ---
 
-It's critical that you deeply understand the problem you're solving. This means that once you understand the problem, you are comfortable exploring and pivoting to different solutions. 
+It's critical that you deeply understand the problem you're solving. This means that once you understand the problem, you are comfortable exploring, evaluating, and pivoting to different solutions.
 
 Just as importantly, you need to bring everyone along with you. Your partners and teammates need to agree on the problem you're tackling and the problems you are not tackling. A well-written problem statement helps align and motivate your team around a common goal, providing focus and keeping distractions at bay.
 
-### Considerations: A good problem statement
```

Comment:
This page (perhaps others, too) jumps from an `h1` to `h3`. For accessibility, the headings should precede sequentially from `h1` to `h2` to `h3` and so on, with `h3`s as "children" of `h2`s.